addon-test-support/scroll.js

```diff
@@ -0,0 +1,47 @@
+import getElementWithAssert from './-private/get-element-with-assert';
+import wait from 'ember-test-helpers/wait';
+
+
+/*
+  Triggers a paint (and therefore flushes any queued up scroll events).
+
+  @method triggerFlushWithPromise
+  @return {RSVP.Promise}
+  @private
+*/
+function triggerFlushWithPromise(){
+  return wait().then(() => {
+    return new Ember.RSVP.Promise((resolve) => {
+      window.requestAnimationFrame(resolve);
+    });
+  });
+}
+
+/*
+  Scrolls DOM element or selector to the given position (if the DOM element is currently overflowed).
+  The promise resolves after the scroll event has been triggered.
+  @method scrollLeft
+  @param {String|HTMLElement} selector
+  @param {Number} position
+  @return {RSVP.Promise}
+  @public
+*/
+export function scrollLeft(selector, position) {
+  getElementWithAssert(selector).scrollLeft = position;
+  return triggerFlushWithPromise();
+}
+
+/*
+  Scrolls DOM element or selector to the given position (if the DOM element is currently overflowed).
+  The promise resolves after the scroll event has been triggered.
+
+  @method scrollTop
+  @param {String|HTMLElement} selector
+  @param {Number} position
+  @return {RSVP.Promise}
+  @public
+*/
```

Comment:
I'm thinking that a single `scrollTo` (like `window.scrollTo`) method that takes X and Y coordinates would be simpler. I think that it's not so uncommon to want to scroll both axis and wait, and with separate methods users would have to do

```
let scrollYPromise = scrollTop('.container', 500);
let scrollXPromise = scrollLeft('.container', 200);
await Ember.RSVP.all(scrollYPromise, scrollXPromise);
```

I don't feel very strong about this tho, but I think that `await scrollTo(200, 500);` is conceptually simpler that having two methods, even if most of the time scroll is only vertical.